### DISCUSSION TITLE
Change scr in position information

### DISCUSSION THREAD

**Discussion by sigsdis30** [opengisch/QField#5181] [Q&A]
I am trying (unsuccesfully) to change wg84 decimal to DMS in my project position information. 
what I've tried so far:
- to change the CRS of my project in project properties
- to change the dispaly of coordinates and angles (general tab)
the position information stay in wgs84 decimal. Did I miss something? Any idea ?

**Comment by SeqLaz** [comment]
Could you try with the following expression?

`to_dms( x(transform($geometry, layer_property(@layer, 'crs'),'EPSG:4326')), 'x', 4, 'aligned')`

**Reply by sigsdis30** [reply]
Thanks for your response. I get this '3°18′45,5285″E' but I am not sure what to do with that. I want to see DMS in the position information in the qfield UI like there: https://docs.qfield.org/how-to/gnss/#gallery-2 on the bottom left corner . I don't want to calculate a specific coordinate. May be I misunderstood something.
